notebook2/notebook/static/styles/buttons.scss

```diff
@@ -0,0 +1,48 @@
+@import './variables/colors.scss';
+#theme-site {
+  a {
+    color: $black;
+    text-decoration: none;
+    &.active,
+    &:hover {
+      color: $devil-gray;
+    }
+  }
+  button {
+    outline: none;
+    /* border: none; */
+    /* display: inline-block; */
+    /* width: 20px; */
```

Comment:
Thoughts on these commented out rules?

Reply:
Copypasta from app.hail.is, need to clean up.